src/routes.tsx

```diff
@@ -16,8 +18,25 @@ import {
   getSolongWallet,
   getTorusWallet,
 } from "@solana/wallet-adapter-wallets";
+import { AccountInformation } from "./views/accountInformation";
+import { TokenMinting } from "./views/token-minting";
+import { SendSolana } from "./views/send-solana";
+import {
+  ConnectionSettingsProvider,
+  useConnectionSettings,
+} from "./contexts/connectionSettings";
+import { TokensProvider } from "./contexts/tokenMap";
+
+export function RoutesWrapper() {
+  return (
+    <ConnectionSettingsProvider>
+      <Routes />
+    </ConnectionSettingsProvider>
+  );
+}
 
-export function Routes() {
+function Routes() {
```

Comment:
Suggestion: `RoutesWrapper` -> `Routes`, `Routes` -> `RoutesWithConnectionSettings`.

The external interface should be `Routes`, the fact that this requires a nested context can be an internal implementation detail of this file.